src/components/menu-item/readme.md

```diff
@@ -21,6 +21,11 @@ The `Menu` component provides container for displaying a list of navigational or
 - **Type:** `string`
 - **Description:** Additional custom classes to style the Menu.
 
+### `collapsed`
```

Comment:
Didn't we discussed this control should be added from Sidebar, than the actual menu item?
@BabuTechBsf

Reply:
collapse is happening from sidebar only, we are just sending this prop to manage inside elements.
Yes we discussed that, but as we know most of the Elements don't use two components they just manipulate using props. So we already providing collapse state from sidebar so they can manage. 
Similar to what we discussed the situation is haldled with the collapse prop.

like
if(collapse)
 collapseComp
 else{
 normalComp
 }

Reply:
That way we will have to manage this for all elements, ex: Menu item.
Let's implement what was discussed earlier.